### PR TITLE
feat(switch): added reverse layout

### PR DESCRIPTION
**Pull request by CooperRedhat**
CSS changes to prepare for [#6059](https://github.com/patternfly/patternfly-react/issues/6059)

### CODE REVIEW

File: src/patternfly/components/Switch/switch.scss [hunk 1]
```diff
@@ -155,7 +167,6 @@
 
 .pf-c-switch__label {
   display: inline-block;
-  grid-column: 2;
```
Comment (mcoker) [inline src/patternfly/components/Switch/switch.scss:158]: I think we talked about leaving the existing CSS in place so as not to break something someone might be relying on the existing CSS as it is currently. I believe it was something like this below, please correct me if I'm wrong!

```
.pf-c-switch {
  &.pf-m-reverse {
    .pf-c-switch__label,
    .pf-c-switch__toggle {
      grid-row: 1;
    }
    .pf-c-switch__label {
      grid-column: 1;
    }
  }
}
```

Comment (CooperRedhat) [reply src/patternfly/components/Switch/switch.scss]: Yes @mcoker, you are correct, patched.

File: src/patternfly/components/Switch/examples/Switch.md [hunk 2]
```diff
@@ -23,6 +23,24 @@ cssPrefix: pf-c-switch
 {{/switch}}
 ```
 
+### Reverse (Toggle on right)
```
Comment (mcoker) [inline src/patternfly/components/Switch/examples/Switch.md:26]: Small nit, we use sentence case pretty much everywhere. 

```suggestion
### Reverse (toggle on right)
```

FYI I did a search to see how we're handling this in other examples and looks like it's lower case.

**Review by mcoker — CHANGES_REQUESTED** [review]
LGTM! Just one small nit on the docs case that I missed before.

**Review by mcoker — APPROVED** [review]
LGTM!

File: src/patternfly/components/Switch/examples/Switch.md [hunk 3]
```diff
@@ -88,3 +106,4 @@ Use checkbox if your user has to perform additional steps for changes to become
 | `.pf-c-switch__label` | `<span>` |  Initiates a label inside the switch. |
 | `.pf-m-on` | `.pf-c-switch__label` | Modifies the switch label to display the on message. |
 | `.pf-m-off` | `.pf-c-switch__label` | Modifies the switch label to display the off message. |
+| `.pf-m-reverse` | `.pf-c-switch` | Modifies the switch toggle show it to the right of the label. |
```
Comment (srambach) [inline src/patternfly/components/Switch/examples/Switch.md:109]: "to show it" :)

**Comment by srambach** [comment]
Looks good other than one typo in the docs! 👍

File: src/patternfly/components/Switch/examples/Switch.md [hunk 4]
```diff
@@ -88,3 +106,4 @@ Use checkbox if your user has to perform additional steps for changes to become
 | `.pf-c-switch__label` | `<span>` |  Initiates a label inside the switch. |
 | `.pf-m-on` | `.pf-c-switch__label` | Modifies the switch label to display the on message. |
 | `.pf-m-off` | `.pf-c-switch__label` | Modifies the switch label to display the off message. |
+| `.pf-m-reverse` | `.pf-c-switch` | Modifies the switch toggle to show it to the right of the label. |
```
Comment (mattnolting) [inline src/patternfly/components/Switch/examples/Switch.md:109]:
```suggestion
| `.pf-m-reverse` | `.pf-c-switch` | Positions the switch toggle to the right of the label. |
```

**Review by mattnolting — CHANGES_REQUESTED** [review]
Looks good, just the one comment

**Review by srambach — APPROVED** [review]
Great job! 🎉

**Review by mattnolting — APPROVED** [review]
Great work, LPTM!

**Pull request merged**